hw6_1

```diff
@@ -0,0 +1,46 @@
+#include<iostream>
+#include<iomanip>
+using namespace std;
+int main(){
+    const int length=500;//length for array length
+    int array[length]={0};//array for store the player's performance
+    for(int i=0;i<length;i++)
+        array[i]=90;
```

Comment:
because of the result of while(b) not always correct, I need to initialize all element in array 90